### PR TITLE
some Alan called, wanted poetry on Ravens

### PR DESCRIPTION
**Pull request by Kristusha100**

### CODE REVIEW

**Comment by gbezyuk** [comment]
It's nice relating #1!

**Review by Kristusha100 — COMMENTED** [review]
I like the poetry, but I don't like the way he calles William names =(

File: poetry/tyger_by_william_blake.txt [hunk 1]
```diff
@@ -30,3 +30,7 @@ Tyger Tyger burning bright,
 In the forests of the night: 
 What immortal hand or eye,
 Dare frame thy fearful symmetry?
+
+
+Critics:
+- bullshit! better write about crows! and, you know, MORE!!! (sincerely yours Edgar Alan Poe)
```
Comment (gbezyuk) [inline poetry/tyger_by_william_blake.txt]: Yeah, man, that's not nice =(

Comment (gbezyuk) [reply poetry/tyger_by_william_blake.txt]: Now that's better

**Pull request merged**